### ISSUE TITLE
Disable or remove esc touch

### ISSUE THREAD

**Issue by asyd** [HiKay/TouchBarDisabler#3]
Hi, 

dunno if it's the right place to ask that, (regarding the project name) but I don't want disable the full touchbar, I just want remove (or move) the esc virtual key? 

Do you think you can manage that?

Thanks anyway

**Comment by HiKay** [comment]
I have made a special fork of TouchBarDisabler just for you! It's called EscapeDisabler and it only removes the escape key on the left and keeps app-based dynamic controls. I've attached it alongside with this email. It does has the side effect of removing brightness/volume controls on the right, but those are remapped to Ctrl+1, Ctrl+2, as well as Ctrl+8~Ctrl+0.

Feel free to give EscapeDisabler a shot and see if it helps.
[EscapeDisabler.zip](https://github.com/HiKay/TouchBarDisabler/files/1075046/EscapeDisabler.zip)

**Comment by asyd** [comment]
Hmm thanks for you quick feedback!

I just ran the application, then select `disable esc key` from the menu, but the esc touch is still active :/

**Comment by jpcope** [comment]
Came here from http://lifehacker.com/how-to-disable-the-macbook-pro-touch-bar-1795941466

A [forum post on apple](https://discussions.apple.com/message/31369191#31369191) mentions using https://github.com/tekezo/Karabiner-Elements to remap the virtual touchbar key to something innocuous but I would really prefer to only disable the escape key and leave the rest functional.

Awesome work on your main use-case product by the way! Going to give it a shot for a couple of days and see if I can get by without function keys in my day-to-day routine. Cheers!

**Comment by asyd** [comment]
@jpcope have you test the EscapeDisabler.zip? does it work for you?

**Comment by Swiftnesses** [comment]
Where can I download the version with the ESC disable options? Thanks!

**Comment by synnys** [comment]
yeah Where can I download the version with the ESC disable options? Thanks!

**Comment by cherian** [comment]
Where can i down the Esc key disabler?

**Comment by bplus** [comment]
Hello!

I'm also looking for the escape-key-only change.  Is that available?  I don't see any other branches for it. Thank you @LumingYin - with or without the escape-key-only disabling, this is helping make my Mac more useful!

**Comment by outlyer** [comment]
@LumingYin Count me as hoping an Escape-only disabler (re)surfaces as well.

**Comment by virqdroid** [comment]
@LumingYin +1 for the Escape-only disabler.

**Comment by Ricarde** [comment]
Try using BetterTouchTool, 

click on the 'Keyboard' tab ('Select Application' should be set to 'Global'),
click on '+ Add New Shortcut or Key Sequence'
click on the field and press the 'Shift-Escape' keys, a ⇧⎋ should appear
move to the right where it says 'Trigger Predefined Action' and choose: 'Keyboard keys/ESC (Escape key, respects pressed modifier)'

We've just created a substitute for the 'escape' key.  Now we need to disable it.

click on '+ Add New Shortcut or Key Sequence'
click on the field and press the 'Escape' key on the Touchbar, a ⎋ should appear
move to the right where it says 'Trigger Predefined Action' and make sure it reads: No Action

I like this way more than deactivating the entire TouchBar.
